sw/device/lib/gpio.c

```diff
@@ -20,7 +18,8 @@ void gpio_write_bit(unsigned int bit, unsigned int val) {
   } else if (bit < 32) {
     gpio_masked_out_reg = (uint32_t *)GPIO_MASKED_OUT_UPPER(0);
   } else {
-    assert(1 && "bit must be < 32");
+    uart_send_str("bit must be less then 32\n");
```

Comment:
Using uart_send_str isntead of assert changes the behaviour - as it won't abort the execution. As the function is void, there is no way to convey this information to the caller. Is it worth makin this function return bool?